code/hispania/modules/food_and_drinks/food/foods/canned_food.dm

```diff
@@ -0,0 +1,101 @@
+///Bases comida enlatada
+/obj/item/reagent_containers/food/snacks/canned_food
+	name = "canned air"
+	desc = "In the distant future air its expensive."
```

Comment:
![image](https://user-images.githubusercontent.com/55407528/116547586-163a3500-a8c1-11eb-99dd-57f3d340a039.png)
He's canon now.

Reply:
GOD I WISH THAT WERE ME